app/controllers/password_resets_controller.rb

```diff
@@ -52,12 +52,14 @@ def update
     elsif params[:user][:password] != params[:user][:password_confirmation]
       # Password does not match password confirmation
       flash.now[:alert] = I18n.t("password_different_notice")
-    elsif @user.update_attributes(user_params)
-      # Clear the user's social uid if they are switching from a social to a local account
-      @user.update_attribute(:social_uid, nil) if @user.social_uid.present?
-
-      # Mark password as secure
-      @user.update(secure_password: true)
+    elsif @user.without_terms_acceptance { @user.update_attributes(user_params) }
+      @user.without_terms_acceptance {
+        # Clear the user's social uid if they are switching from a social to a local account
+        @user.update_attribute(:social_uid, nil) if @user.social_uid.present?
+
+        # Mark password as secure
+        @user.update(secure_password: true)
+      }
```

Comment:
Is this still needed with https://github.com/bigbluebutton/greenlight/pull/3101 merged?

Reply:
It's probably best to put up a new MR that works with the upstream code

Reply:
Alright, I'll get back to it ASAP and I'll notify you Inshallah.